airbyte-webapp-e2e-tests/cypress/pages/connectorBuilderPage.ts

```diff
@@ -94,3 +123,12 @@ export const testStream = () => {
   cy.wait(500);
   cy.get(testStreamButton).click();
 };
+
+const GO_BACK_AND_GO_NEXT_BUTTONS = 2
+export const assertHasNumberOfPages = (numberOfPages: number) => {
+  for (var i = 0; i < numberOfPages; i++) {
```

Comment:
It seems to me like the best approach for this is to look for the `a` elements instead of the `li`s and don't count `rel="prev"` or `rel="next"` - best way to do it I can see is `cy.get(SELECTOR).not(`[rel="prev"]`).not(`[rel="next"]`).to('have.length'`. I don't think we need to validate the actual contents of those elements.

We can rely on the `rel` attributes here because those are semantic attributes, not just an implementation detail: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel

We still assume a bit about the structure of this component by looking for `a` tags, but I don't see a good way around that and I think that's fairly stable.

Reply:
As the focus is on page 1, there are two elements with "rel=next": page 2 and ">". Therefore, we can't use this solution even though it would have been cleaner